anndata/readwrite/read.py

```diff
@@ -321,6 +321,55 @@ def _read_text(
                    var={'var_names': col_names})
 
 
+def read_zarr(store):
+    import zarr
+    f = zarr.open(store, mode='r')
+    d = {}
+    for key in f.keys():
+        _read_key_value_from_zarr(f, d, key)
+    return AnnData(d)
+
+
+def _read_key_value_from_zarr(f, d, key, key_write=None):
```

Comment:
this is very redundant, instead we should have a helper that is called both by the HDF5 functions and the zarr... @Koncopd what do you think?